src/main/java/org/darisadesigns/polyglotlina/Desktop/PolyGlot.java

```diff
@@ -17,7 +17,7 @@
  * ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
  * POSSIBILITY OF SUCH DAMAGE.
  */
-package org.darisadesigns.polyglotlina;
```

Comment:
Just as a note, this will be functionality I want to do a lot of functional stress testing on. There's code that reloads the core and needs to do housekeeping every time a new PolyGlot file is created, PolyGlot options are changed, and a few other situations.